### PR TITLE
OKTA-479360 Update SCIM Provisioning integration images

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update SCIM Provisioning integration images in "Connect your SCIM API service to Okta". Updated test plan XLS in "Test your Okta SCIM integration".
- **Is this PR related to a Monolith release?** No

### Netlify preview:
Mar 17 (reduced the number of images, updated text): 
   * https://623376b1bce85813be3377ee--reverent-murdock-829d24.netlify.app/docs/guides/scim-provisioning-integration-connect/main/ 
   * https://623376b1bce85813be3377ee--reverent-murdock-829d24.netlify.app/docs/guides/scim-provisioning-integration-test/main/

Mar 15: https://623125a75c2cd651e35d2d9b--reverent-murdock-829d24.netlify.app/docs/guides/scim-provisioning-integration-connect/main/

### Resolves:

* [OKTA-479360](https://oktainc.atlassian.net/browse/OKTA-479360)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 1]
```diff
@@ -91,59 +91,59 @@ This page contains settings for all information that flows from Okta user profil
...
+* **User Creation & Matching**: Sets up matching rules to use when importing users from your application. Establishing matching criteria allows you to specify how an imported user is defined when creating a new user or mapping to an existing Okta user.
 
-  * **Imported user is an exact match to Okta user if** &mdash; sets the matching criteria to determine when an imported user matches to an existing Okta user. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
+  * **Imported user is an exact match to Okta user if**: Sets the matching criteria to determine when an imported user matches to an existing Okta user. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:122]:
```suggestion
  * **Imported user is an exact match to Okta user if**: Sets the matching criteria to determine when an imported user matches to an existing Okta user. To consider the new imported user an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 2]
```diff
@@ -159,84 +159,71 @@ When you add a SCIM template integration to your development org, it comes with
...
 
-  1. After removing the mapping for the unwanted attributes, click **To Okta** under the settings section.
-    ![Provisioning to Okta tab](/img/oin/scim_check-attributes-4.png "Provisioning Tab: Provisioning to Okta")
+  1. Scroll to the **Attribute Mappings** section. Look for the attribute that you want to delete and then click the corresponding **X**.
 
-  Repeat Step 3 and 4 until you remove all the mappings for the attributes you want to delete.
+      ![Displays the Attribute Mappings screen.](/img/oin/scim_check-attributes-2.png "Attribute Mappings")
 
-B. Delete attributes from your attribute list
+  1. Click **OK** to confirm that you want to remove the mapping for the attribute you selected.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:172]:
```suggestion
  1. Click **OK** to confirm that you want to remove the mapping for the attribute that you selected.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 3]
```diff
@@ -159,84 +159,71 @@ When you add a SCIM template integration to your development org, it comes with
...
-  ![Profile Editor - Add Attribute](/img/oin/scim_check-attributes-10.png "Profile Editor - Add Attribute")
-  Enter the information for the new attribute that you’re adding and then click Save. For example:
-  ![Profile Editor - Add Attribute Dialog](/img/oin/scim_check-attributes-11.png "Profile Editor - Add Attribute Dialog")
 
-   >**Note:** The Scope property determines whether the attribute you are adding can be assigned at a group level or just per user. If you would like your admins to be able to assign a value for this attribute at a group level, don't check the **User personal** checkbox.
+1. Enter the information for the new attribute that you’re adding and then click **Save**. For example:
+
+    ![Displays the Add Attribute dialog](/img/oin/scim_check-attributes-11.png "Profile Editor: Add Attribute")
+
+    > **Note:** The Scope property determines whether the attribute you are adding can be assigned at a group level or just per user. If you would like your admins to be able to assign a value for this attribute at a group level, don't check the **User personal** checkbox.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:200]: should Scope be bolded or `Scope`?
```suggestion
    > **Note:** The Scope property determines whether the attribute that you are adding can be assigned at a group level or just per user. If you want your admins to be able to assign a value for this attribute at a group level, don't select the **User personal** checkbox.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 4]
```diff
@@ -159,84 +159,71 @@ When you add a SCIM template integration to your development org, it comes with
...
+1. Go to the **Provisioning** tab. Under the **Settings** section, click **To App**.
 
 1. Scroll to the **Attribute Mappings** section. Look for the attribute that you want to update and click **Edit**.
-  ![Attribute Mappings - Edit Attribute](/img/oin/scim_check-attributes-13.png "Attribute Mappings - Edit Attribute")
 
-1. In the dialog that appears, there are two drop-down fields. In the first drop-down menu, select **Map from Okta Profile**. In the second drop-down menu, choose the Okta profile attribute that you would like to map the SCIM attribute from. Click **Save**.
-  ![Attributes - Map Attribute](/img/oin/scim_check-attributes-14.png "Attributes - Map Attribute")
+    ![Displays the Attribute Mappings > Edit Attribute screen.](/img/oin/scim_check-attributes-13.png "Attribute Mappings: Edit Attribute")
+
+1. In the dialog that appears, there are two dropdown fields. In the first dropdown menu, select **Map from Okta Profile**. In the second dropdown menu, choose the Okta profile attribute that you would like to map the SCIM attribute from. Click **Save**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:214]:
```suggestion
1. In the dialog that appears, there are two dropdown fields. In the first dropdown menu, select **Map from Okta Profile**. In the second dropdown menu, choose the Okta profile attribute that you want to map the SCIM attribute from. Click **Save**.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 5]
```diff
@@ -159,84 +159,71 @@ When you add a SCIM template integration to your development org, it comes with
...
+
+    ![Displays the Map Attribute dialog.](/img/oin/scim_check-attributes-14.png "Attributes: Map Attribute")
 
-1. Repeat for all other SCIM attributes where you would like to modify the mapping (from Okta to your application).
+1. Repeat steps 3 and 4 for all other SCIM attributes you want to map (from Okta to your application).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:218]:
```suggestion
1. Repeat steps 3 and 4 for all other SCIM attributes that you want to map (from Okta to your application).
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 6]
```diff
@@ -159,84 +159,71 @@ When you add a SCIM template integration to your development org, it comes with
...
 
-1. After updating the mappings from Okta to your application, click **To Okta** under the settings section.
-  ![Provisioning to Okta tab](/img/oin/scim_check-attributes-4.png "Provisioning Tab: Provisioning to Okta")
+1. After updating the mappings from Okta to your application, click **To Okta** under the **Settings** section.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:220]:
```suggestion
1. After you update the mappings from Okta to your application, click **To Okta** under the **Settings** section.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md [hunk 7]
```diff
@@ -248,12 +235,12 @@ You only want to include the attributes that you support in your current user sc
...
-    1. Go to your Admin Console.
-    1. Click **Directory** > **People**.
-    ![Admin Dashboard - Directory - People](/img/oin/scim_check-attributes-18.png "Dashboard showing Directory to People menu item")
-    1. You should see the list of Okta users for your org. Find the user you just imported and click that user's name.
-    1. Once the user account is opened, click **Profile**. The Profile screen shows you that user's attributes. Check whether the values for the attributes you support were imported properly for this user.
-    ![User Profile Attributes](/img/oin/scim_check-attributes-19.png "User attributes dialog")
+    1. From your Admin Console, select **Directory** > **People**. The list of Okta users for your org appears.
+
+    1. Find the user that you imported and click the user's name.
+    1. Once the user account appears, click **Profile**. The Profile screen shows the user's attributes. Verify that the supported attribute values were imported properly.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:241]:
```suggestion
    1. After the user account appears, click **Profile**. The Profile screen shows the user's attributes. Verify that the supported attribute values were imported properly.
```

**Review by susanharper-okta — APPROVED** [review]
couple nits. looking good. approving to not hold you up.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md (hunk 3 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:200]: Yes, Scope needs to be bolded. Thanks!

**Review by andreiiatiniuc-okta — COMMENTED** [review]
LGTM

**Pull request merged**